test/integration/query-interface.test.js

```diff
@@ -35,22 +35,22 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
 
   describe('showAllTables', () => {
     it('should not contain views', async function() {
-      async function cleanup() {
+      async function cleanup(sequelize) {
         // NOTE: The syntax "DROP VIEW [IF EXISTS]"" is not part of the standard
         // and might not be available on all RDBMSs. Therefore "DROP VIEW" is
         // the compatible option, which can throw an error in case the VIEW does
         // not exist. In case of error, it is ignored.
         try {
-          await this.sequelize.query('DROP VIEW V_Fail');
+          await sequelize.query('DROP VIEW IF EXISTS V_Fail');
         } catch (error) {
           // Ignore error.
         }
```

Comment:
Would this work fine if the try/catch block was removed?

Reply:
It should work, try/catch expression might be removed